### PR TITLE
GTH pp interface to Sirius

### PR DESCRIPTION
**Pull request by juerghutter**

### CODE REVIEW

File: src/sirius_interface.F [hunk 1]
```diff
@@ -224,11 +238,152 @@ SUBROUTINE cp_sirius_create_env(pwdft_env)
...
+         NULLIFY (upf_pot, gth_potential)
+         CALL get_qs_kind(qs_kind_set(ikind), upf_potential=upf_pot, gth_potential=gth_potential)
+         IF (ASSOCIATED(upf_pot)) THEN
+            CALL sirius_add_atom_type(sctx, string(label), fname=string(upf_pot%filename), &
+                                      mass=REAL(mass, KIND=C_DOUBLE))
+         ELSEIF (ASSOCIATED(gth_potential)) THEN
+            !
+            NULLIFY (atom_grid)
+            CALL allocate_grid_atom(atom_grid)
+            nmesh = 929
```
Comment (dev-zero) [inline src/sirius_interface.F:252]: accidentally hardcoded number of grid points?

Comment (juerghutter) [reply src/sirius_interface.F:252]: No. We didn't wanted to make this an input option, just use a hardcoded default.

**Comment by oschuett** [comment]
I see that you upper-cased the regtest directory in to `/tests/SIRIUS/` (good!). I guess, then `TEST_DIRS`  should be updated and the old lower-case dir removed.

**Comment by oschuett** [comment]
I think `tests/TEST_DIRS` still needs to be updated.

**Pull request merged**